### ISSUE TITLE
characters overlay when mouse hover the terminal

### ISSUE THREAD

**Issue by piLeoni** [blessed-ng/blessed#7]
Hi all, i'm trying to test a bit blessed but very often it happen that when i move my mouse on the terminal an amount of character are printed from the top of the screen covering all the rest:

<img width="697" alt="screen shot 2018-06-12 at 17 55 04" src="https://user-images.githubusercontent.com/3159836/41340653-2b38d8e0-6ef8-11e8-989e-ca034d0e43c0.png">

**Comment by piranna** [comment]
Seems they are the ANSI escape codes of the mouse events.

**Comment by piLeoni** [comment]
Yes it seems like but this is just the basic usage example suggested on the github page, why this happen?

**Comment by piranna** [comment]
Maybe configuration of your terminal, or you need to enable mouse support on your application.

**Comment by piLeoni** [comment]
May be this problem:
https://github.com/remy/nodemon/issues/890

**Comment by Trackhe** [comment]
Any Solution?
..
I use https://github.com/chjj/blessed/issues/334#issuecomment-381405970
